classes/opponentPhaseBullethell3.lua

```diff
@@ -0,0 +1,101 @@
+-- The template opponent phase
+
+local opponentPhaseBullethell3 = {}
+opponentPhaseBullethell3.__index = opponentPhaseBullethell3
+
+
+local ORBIT_START_ANG = -math.pi / 2
+local ORBIT_ANG_SPEED = 1.2 	-- How many radians per second should the opponent advance in orbit?
+local ORBIT_RADIUS_X = 400
+local ORBIT_RADIUS_Y = 250
+
+local MAX_TURN_SPEED = 1
+
+local ORBIT_FOCAL_X = ARENA_WIDTH / 2
+local ORBIT_FOCAL_Y = ARENA_HEIGHT / 2
+
+opponentPhaseBullethell3.RADIUS = 55
```

Comment:
he T H I C C

I reduced the size of phase 2 in my tweaks. This might want to be reduced as well